src/ServiceProvider.php

```diff
@@ -91,6 +111,23 @@ protected function loadMigrations()
         $this->loadMigrationsFrom(__DIR__ . '/../database/migrations');
     }
 
+    /**
+     * Returns existing migration file if found, else uses the current timestamp.
+     *
+     * @param Filesystem $filesystem
+     * @return string
+     */
+    protected function getMigrationFileName($filesystem): string
```

Comment:
Could you please explain what is this for?

Reply:
I create a stub file (create_saml2_tenants_table.stub) of the migration and when executing the command to publish, a timestamp will be created with now().

Reply:
Yeah, I think I got that :) 

Why is it necessary?

Reply:
The main objective is to prevent duplication, we know that this is very rare to happen with this table name, but, better to prevent.

Reply:
No problem. Done!